src/controller.py

```diff
@@ -107,6 +107,9 @@ def run_command_and_get_its_response(self) -> dict:
         elif self.command == "emulator-read-and-confirm-mnemonic":
             emulator.read_and_confirm_mnemonic()
             return {"response": "Read and confirm mnemonic"}
+        elif self.command == "emulator-read-and-confirm-mnemonic-shamir":
```

Comment:
Maybe `emulator-read-and-confirm-shamir-mnemonic` better? Or `emulator-read-and-confirm-slip39-mnemonic`?

Reply:
I like the first one, `emulator-read-and-confirm-shamir-mnemonic`